go/cache/lru_cache_test.go

```diff
@@ -102,13 +97,11 @@ func TestDelete(t *testing.T) {
 	cache.Set(key, value)
 	cache.Delete(key)
 
-	if sz := cache.UsedCapacity(); sz != 0 {
-		t.Errorf("cache.UsedCapacity() = %v, expected 0", sz)
-	}
+	sz := cache.UsedCapacity()
+	assert.Zero(t, sz, "cache.UsedCapacity() = %v, expected 0", sz)
```

Comment:
There is no need to report `sz`, since `assert.Zero(...)` will already report the value if nonzero.
```suggestion
	assert.Zero(t, sz)
```